web/src/main/java/org/openmrs/web/controller/form/FormResourceController.java

```diff
@@ -10,12 +10,15 @@
 package org.openmrs.web.controller.form;
 
 import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
 
 import org.apache.commons.lang.StringUtils;
+import org.jfree.util.Log;
```

Comment:
This is wrong. Please see other OpenMRS code to see the standard way we log, by convention (e.g. it uses log4j, not jfree.util).